torchtitan/experiments/README.md

```diff
@@ -0,0 +1,18 @@
+## Contributing Guidelines
+
+We provide this `experiments/` folder to host experiments that add significant value to `torchtitan`, with the following principles. We refer to the part of `torchtitan` outside `experiments` as `core`.
+1. Each subfolder in `experiments` will be an experiment, with a clear theme which can be flexible, such as
+    - a new model, or preferably a new model architecture, with its training infrastructure including parallelization functions;
+    - an enhancement or addition to the existing infrastructures in `torchtitan`.
+2. It is the contributors' responsibility to justify the value of an experiment. `torchtitan` team will review proposals on a case-by-case basis. As part of the contribution, the contributors should provide documentation that clearly showcases the motivation and innovation of an experiment, including reports on performance and loss convergence.
+3. An experiment should reuse existing `torchtitan` code as much as possible, such as modules in [`components/`](../components/) (via a new [`TrainSpec`](../protocols/train_spec.py)) and [`train.py`](../train.py). For a list of extension points we provide, please refer to [docs/extension.md](../../docs/extension.md).
+    - The extension points are subject to change. Contributors should provide feedback if they find troubles reusing the components, instead of simply making copy-and-paste.
```

Comment:
re: "Contributors should provide feedback if they find troubles reusing the components, instead of simply making copy-and-paste."
smoother:
"We kindly request that contributors provide feedback if they encounter issues re-using any components, rather than simply using a copy and paste approach. "